src/initialize-test-window.coffee

```diff
@@ -57,6 +57,12 @@ module.exports = ({blobStore}) ->
 
     document.title = "Spec Suite"
 
+    # Avoid throttling of test window by playing silence
+    context = new AudioContext()
+    source = context.createBufferSource()
+    source.connect(context.destination)
+    source.start(0)
+
     testRunner = require(testRunnerPath)
```

Comment:
@nathansobo Any idea if this trick works as far back as 0.29.2? We're having odd throttling behavior (networking and cpu) currently but can't upgrade our Electron because we're having build issues for our ARM environment.

We're working on getting unblocked on upgrading, but in the meantime any techniques to disable all throttling behavior in chromium is desired. Related, any advice on making Chrome only perform at maximum would be helpful. We are adding our own resource throttling at a higher level in our custom linux build.